fdbserver/storageserver.actor.cpp

```diff
@@ -14745,6 +14746,7 @@ ACTOR Future<Void> storageServer(IKeyValueStore* persistentData,
 		self.setTssPair(ssi.tssPairID.get());
 		ASSERT(self.isTss());
 	}
+	static_assert(sizeof(self) < 16384, "FastAlloc doesn't allow allocations larger than 16KB");
```

Comment:
Ideally the static assert would apply to all FastAllocated types, and/or FastAllocated should handle > 16384 correctly. Not handling it correctly was a mistake

Reply:
Agree.
debug build has the assertion https://github.com/jzhou77/foundationdb/blob/fix/flow/include/flow/FastAlloc.h#L237, the problem is we don't usually test with debug build.